static/script.js

```diff
@@ -11,8 +11,37 @@ $(function () {
   })
 
   $('.clickable').click(function () {
-    $(this).toggleClass('unfold')
-    $(this).parent().toggleClass('active-border')
+    let $item = $(this)
+
+    let allUnfoldIds = sessionStorage.getItem('unfoldedRows')
+    let rowId = $item.attr('id')
+    let newSessionStorageValue = ''
+
+    $item.toggleClass('unfold')
+    $item.parent().toggleClass('active-border')
+
+    rowId = rowId.replace('heading', 'collapse')
+
+    if (allUnfoldIds === null) {
+      newSessionStorageValue = rowId + ','
+    } else {
+      let ids = allUnfoldIds.split(',')
+      let currentIdIndex = ids.indexOf(rowId)
+      if (currentIdIndex === -1) {
+        newSessionStorageValue = allUnfoldIds + rowId + ','
```

Comment:
IMO this whole If-else statement can be shorten to something like this:
```javascript
let allUnfoldIds = sessionStorage.getItem('unfoldedRows')
let rowId = $item.attr('id')
let newSessionStorageValue = []

if (allUnfoldIds === null) {
    newSessionStorageValue.push(rowId)
} else {
   let ids = allUnfoldIds.split(",")
   newSessionStorageValue = ids
   if (! ids.contains(rowId)) {
      newSessionStorageValue.push(rowId)
   }
}
sessionStorage.setItem('unfoldedRows', newSessionStorageValue.join(","))
```
Always when possbile use array split/join instead of concatenaiting strings.